hack/aks/Makefile

```diff
@@ -10,7 +10,7 @@ AZCLI   ?= docker run --rm -v $(AZCFG):/root/.azure -v $(KUBECFG):/root/.kube -v
 # overrideable defaults
 REGION     ?= westus2
 OS_SKU     ?= Ubuntu
-VM_SIZE	   ?= Standard_B2s
+VM_SIZE	   ?= Standard_DS3_v2
```

Comment:
i don't think the default vm or lb skus should be changed, these are way more expensive. if we need bigger nodes for say a load test, then that test should override this via the env var

Reply:
tbh I'm not opinionated about the changes in this pr, I'm just reusing the pipeline for quick clusters at the moment :)